### PR TITLE
Weight info tooltip

### PR DESCRIPTION
**Pull request by singhalex**
Mouseover the icon to get info on how the weight works

### CODE REVIEW

**Review by belunatic — APPROVED** [review]

File: expense-splitter/src/components/expense/CreateExpense.jsx [hunk 1]
```diff
@@ -346,7 +350,15 @@ export default function CreateExpense() {
         <div className="mb-8">
           {watchedValues["group"] && (
             <>
-              <h2 className="mb-4">Weight Contributions:*</h2>
+              <h2 className="mb-4">
+                Weight Contributions:*
+                <Tooltip title={weightInfo} placement="right">
+                  <i
```
Comment (ccespedes) [inline expense-splitter/src/components/expense/CreateExpense.jsx]: In mobile view, the message is cut off. Try placement="bottom-end"

File: expense-splitter/src/components/expense/CreateExpense.jsx [hunk 2]
```diff
@@ -346,7 +350,15 @@ export default function CreateExpense() {
...
+                    className="fa-solid fa-circle-info ml-1"
+                    style={{ color: "#72a1e5" }}
```
Comment (ccespedes) [inline expense-splitter/src/components/expense/CreateExpense.jsx]: Remove style and update classname to:
className="fa-solid fa-circle-info ml-1 text-accent"

**Review by ccespedes — COMMENTED** [review]
Seems good but checkout the comments.

File: expense-splitter/src/components/expense/CreateExpense.jsx (hunk 1 above)
Comment (singhalex) [reply expense-splitter/src/components/expense/CreateExpense.jsx]: I forgot to test mobile view XD

I can't trigger it on mobile view for some reason ...

Comment (singhalex) [reply expense-splitter/src/components/expense/CreateExpense.jsx]: And of course I figured it out as soon as I wrote that

**Review by ccespedes — APPROVED** [review]

**Comment by ccespedes** [comment]
Muahaha 🤣

**Pull request merged**